test-tunnel-client.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Test Tunnel MCP Client
+echo "🧪 Testing Tunnel MCP Client..."
+
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)